### DISCUSSION TITLE
Can the google.golang.org/grpc/status package be used to get error codes on the client when using a buf.connect server?

### DISCUSSION THREAD

**Discussion by varunbpatil** [connectrpc/connect-go#557] [Q&A]
Hi, I have a server that uses connect-go. But, my client can only use a grpc client (not a connect client). Is it possible to get a `google.golang.org/grpc/status` error on the client using something like `status.FromError(err)`? Note that I can't import any connect components on the client.

**Comment by akshayjshah** [comment]
Yes! The data on the wire will be in the gRPC wire format, so the client has no idea that the server is built with Connect. Clients can can retrieve a protobuf Status message exactly as they usually would.
